### DISCUSSION TITLE
Just here to say I LOVE this theme!

### DISCUSSION THREAD

**Discussion by freddiehaddad** [EdenEast/nightfox.nvim#377] [General]
Great work on this theme, especially the plugin design for neovim.

I'm not only using it there, but also applied the palette to:

- fzf
- bat
- wofi
- waybar
- sway
- tmux

Love it!

**Comment by nicos68** [comment]
Hey ! If you have the config files in a repo somewhere I’d be interested to have a look 👀 (I had a quick look on your dotfiles but couldn’t find the bat theme)

**Reply by freddiehaddad** [reply]
Hi,

I presume you looked at this repo https://github.com/freddiehaddad/dotfiles?

`bat` is using the default terminal theme which is set to Nightfox palette.

Alacritty: https://github.com/freddiehaddad/dotfiles/blob/b734eb42f750ef3209e48db739b4e7f6020678c4/home/freddiehaddad/.config/alacritty/alacritty.toml

In my `.zshrc`:
https://github.com/freddiehaddad/dotfiles/blob/b734eb42f750ef3209e48db739b4e7f6020678c4/home/freddiehaddad/.zshrc#L93C1-L97C1

```zsh
# Better cat
if (($+commands[bat])); then
	alias cat="bat --plain --theme ansi"
fi
```

I also set dircolors (it might be picking up the colors from that if Alacritty isn't doing it):
https://github.com/freddiehaddad/dotfiles/blob/b734eb42f750ef3209e48db739b4e7f6020678c4/home/freddiehaddad/.config/dircolors

**Reply by nicos68** [reply]
okay, it might be because of dircolors(first time I hear about this), because in my case, I’m using catppuccin as my dark theme (and dayfox as my light teme) 
![image](https://github.com/user-attachments/assets/1bd677d2-da48-4073-a200-9f453ba19880)

and with the `--theme ansi` option setup, the result is far inferior:
![image](https://github.com/user-attachments/assets/d8af0b3c-9923-4d25-9b9e-1c9e196f2580)
(see my dotfiles [here](https://codeberg.org/nise/dotfiles) if you’re interested, bat config [here](https://codeberg.org/nise/dotfiles/src/branch/main/.config/bat))

Also, I’m using bash, not zsh.
